app/models/ems_event/automate.rb

```diff
@@ -12,6 +12,18 @@ def refresh(*targets, sync)
       EmsRefresh.queue_refresh(refresh_targets, nil, sync)
     end
 
+    def refresh_new_target
+      ems = ext_management_system
+      if ems.supports_refresh_new_target?
+        event_data = {
+          :full_data => full_data,
+          :message   => message,
+          :ems_id    => ems_id
+        }
+        EmsRefresh.queue_event_for_refresh(event_data, ems)
+      end
```

Comment:
And if it doesn't support `refresh_new_target` it should probably elevate to a full ems refresh rather than just skipping it